controller/workitem.go

```diff
@@ -111,6 +112,11 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	if !authorized {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("user is not authorized to access the space"))
 	}
+	// load the current version of the work item.
+	oldWi, err := c.db.WorkItems().LoadByID(ctx.Context, wi.ID)
```

Comment:
Aren't we already loading that work item a few lines up, here https://github.com/fabric8-services/fabric8-wit/pull/2154/files#diff-02845b093c442bf1ae790a6ebb6c8da8R98 ?

Reply:
We can use https://github.com/jinzhu/copier for that. You're trading performance for convenience.

Reply:
@michaelkleinhenz I agree with @kwk I'm unsure how this is more convenient..?